environment/src/main/java/com/sequenceiq/environment/experience/liftie/LiftieResponseReader.java

```diff
@@ -0,0 +1,42 @@
+package com.sequenceiq.environment.experience.liftie;
+
+import static com.sequenceiq.cloudbreak.util.NullUtil.throwIfNull;
+import static javax.ws.rs.core.Response.Status.Family.SUCCESSFUL;
+
+import java.util.Optional;
+
+import javax.ws.rs.ProcessingException;
+import javax.ws.rs.core.Response;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.environment.experience.ResponseReader;
+
+@Component
+public class LiftieResponseReader implements ResponseReader {
```

Comment:
Only the way of logging. Basically, we're doing exactly the same thing here and there but to have more refined logging (since we're executing an operation on a resource that comes from a completely different source (common xp vs liftie based xp) that can be searched more precisely during some investigation

Reply:
ok, I hope we won't forget to update all of these classes